README.md

```diff
@@ -96,4 +96,526 @@ On the joining screen, choose to join the meeting without `CallKit` or join via
+    - [Use case 12. Start viewing local video tile](#use-case-12-start-viewing-local-video-tile)
+    - [Use case 13. Stop sharing your video.](#use-case-13-stop-sharing-your-video)
+    - [Use case 14. Stop viewing local video](#use-case-14-stop-viewing-local-video)
+    - [Use case 15. View only one remote attendee video. e.g. 1-on-1 session. Tutor-Student session](#use-case-15-view-only-one-remote-attendee-video-eg-1-on-1-session-tutor-student-session)
+  - [Screen share](#screen-share)
+    - [Use case 16. Start/Stop viewing remote screen share.](#use-case-16-startstop-viewing-remote-screen-share)
+  - [Metrics](#metrics)
+    - [Use case 17. Start receiving metrics](#use-case-17-start-receiving-metrics)
+  - [Data Message](#data-message)
+    - [Use case 18. Start receiving data message](#use-case-18-start-receiving-data-message)
+    - [Use case 19. Start sending data message](#use-case-19-start-sending-data-message)
+  - [Stopping a session](#stopping-a-session)
+    - [Use case 20. Stop session](#use-case-20-stop-session)
+  - [Voice Focus](#voice-focus)
+    - [Use case 21. Enable/Disable voice focus](#use-case-21-enabledisable-voice-focus)
+  - [Custom Video Source](#custom-video-source)
+    - [Use case 22. Enable/Disable torch (back light)](#use-case-22-enabledisable-torch-back-light)
+## Starting a session
+
+### Use case 1. Start a session. 
+
+To start hearing audio and receiving video, you’ll just need to start the session.
+
+```
+currentMeetingSession.audioVideo.start()
+```
+
+### Use case 2. Add an observer to receive audio and video session life cycle events. 
+
+```
+class MeetingModel : AudioVideoObserver {
+    **func** audioSessionDidStartConnecting(reconnecting: Bool) {}
+    **func** audioSessionDidStart(reconnecting: Bool) {
+        // Handle mute of self attendee.
+    }
+    **func** audioSessionDidDrop() {}
+    **func** audioSessionDidStopWithStatus(sessionStatus: MeetingSessionStatus) {}
+    **func** audioSessionDidCancelReconnect() {}
+    **func** connectionDidRecover() {}
+    **func** connectionDidBecomePoor() {}
+    **func** videoSessionDidStartConnecting() {}
+    **func** videoSessionDidStartWithStatus(sessionStatus: MeetingSessionStatus) {}
+    **func** videoSessionDidStopWithStatus(sessionStatus: MeetingSessionStatus) {}
+    
+    currentMeetingSession.audioVideo.addAudioVideoObserver(observer: **self**)
+}
+```
+
+## Device
+
+### Use case 3. List audio/video devices
+
+List available devices that can be used for the meeting
+
+```
+let devices = currentMeetingSession.audioVideo.listAudioDevices()
+```
+
+### Use case 4. Choose audio device by passing `MediaDevice` object
+
+> NOTE: chooseAudioDevice is no-op if it is called before audioVideo.start(). You should call this after audio session has started. You can put it in audioSessionDidStart callback
+
+```
+let devices = audioVideo.listAudioDevices()
+// Have your own logic to sort listAudioDevices() 
```

Comment:
Do we need to mention the sorting? I think it's not related to the use case though, maybe we can do something like [JS](https://github.com/aws/amazon-chime-sdk-js/blob/master/README.md#device) Use case 2.